terraform/fastly-exporter/main.tf

```diff
@@ -0,0 +1,109 @@
+locals {
+  name = "fastly-exporter"
+}
+
+data "aws_ssm_parameter" "fastly_api_token" {
+  name = "/prod/crates-io/fastly/api-token"
+}
+
+resource "aws_iam_policy" "read_fastly_api_token" {
+  name = "ecs--${local.name}"
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Sid      = "AllowReadingFastlyApiToken"
+        Effect   = "Allow"
+        Action   = "ssm:GetParameters"
+        Resource = data.aws_ssm_parameter.fastly_api_token.arn
+      }
+    ]
+  })
+}
+
+resource "aws_iam_role_policy_attachment" "read_fastly_api_token" {
+  role       = module.ecs_task.execution_role_name
+  policy_arn = aws_iam_policy.read_fastly_api_token.arn
+}
+
+module "ecs_task" {
+  source = "../shared/modules/ecs-task"
+
+  name   = local.name
+  cpu    = 256
+  memory = 512
+
+  log_retention_days = 7
+  ecr_repositories_arns = [
+    # This repository does not exist, since we're pulling the imge directly from GitHub.
+    # But the task module cannot be applied without providing at least one ARN here.
+    "arn:aws:ecr:us-west-1:890664054962:repository/fastly-exporter"
+  ]
+
+  containers = <<EOF
```

Comment:
Doesn't matter that much, but it'd be nice to use `jsonencode()` here rather than plaintext JSON.

Reply:
We have more blocks like this in Terraform. Let me prepare a separate PR to switch them to `jsonencode`.